tjreports/administrator/views/reports/view.html.php

```diff
@@ -100,9 +100,11 @@ public function display($tpl = null)
 		}
 
 		$input->set('reportId', $reportId);
+		$filterData  = $input->get('filters', '', 'ARRAY');
+		$model = $this->getModel();
 
 		// Get respected plugin data
-		$this->items		= $this->get('Data');
+		$this->items		= $model->getData($filterData);
```

Comment:
@ranjit-redekar  in model, the filters should have been set. You should not get any vars in view.html.php